packages/card/index.js

```diff
@@ -0,0 +1,116 @@
+import { FabricWebComponent } from '../utils';
+import { card as c } from '@fabric-ds/component-classes';
+import { classNames } from '@chbphone55/classnames';
+
+class FabricCard extends FabricWebComponent {
+  static get observedAttributes() {
+    return ['onlick', 'selected'];
```

Comment:
Tehehehehe. You see it? You see what I’m chuckling at? Best typo ever!

Reply:
This is not a bug, this is a feature! 😆